src/Temporalio/Client/WorkflowOptions.cs

```diff
@@ -62,6 +62,15 @@ public WorkflowOptions(string id, string taskQueue)
         /// </summary>
         public WorkflowIdReusePolicy IdReusePolicy { get; set; } = WorkflowIdReusePolicy.AllowDuplicate;
 
+        /// <summary>
+        /// Gets or sets how already-existing workflows of the same ID are treated. Default is
+        /// <see cref="WorkflowIdConflictPolicy.Unspecified" /> which effectively means
+        /// <see cref="WorkflowIdConflictPolicy.Fail"/> on the server. If this value is set, then
```

Comment:
Same comment as Python. Maybe we just want to default to FAIL explicitly?

Reply:
:+1: Discussed at https://github.com/temporalio/sdk-python/pull/579#discussion_r1676299240 (basically it's unfortunate but required)